rdt/transformers/null.py

```diff
@@ -114,6 +117,10 @@ def fit(self, data):
             if self._missing_value_generation == 'random':
                 self._null_percentage = null_values.sum() / len(data)
 
+        if self._missing_value_replacement == 'random':
+            self._min_value = data.min()
+            self._max_value = data.max()
```

Comment:
Kind of an edge case, but what happens if all the values are nan? I think the min and max would be set to nan as well.

Reply:
If all values are nan we do `0` 
```python
        if self._missing_value_replacement in {'mean', 'mode', 'random'} and pd.isna(data).all():
            msg = (
                f"'missing_value_replacement' cannot be set to '{self._missing_value_replacement}'"
                ' when the provided data only contains NaNs. Using 0 instead.'
            )
            LOGGER.info(msg)
            return 0

```